src/main/java/commands/diet/dietmanager/DietSessionCreate.java

```diff
@@ -21,9 +20,20 @@ public void execute(String input, Storage storage) {
             assert ds != null;
             logger.log(Level.INFO, "Diet session successfully created");
             ds.start();
+            saveToFile(storage, ds);
         } catch (IOException e) {
             System.out.println("It seems like we ran into some problems saving your session...");
             logger.log(Level.WARNING, "Diet session failed to save");
         }
     }
+
+    private void saveToFile(Storage storage, DietSession ds) {
+        try {
+            storage.init(ds.getTypeInput() + " " + ds.getDate().toString());
+            storage.writeToStorageDietSession(ds.getTypeInput() + " " + ds.getDate().toString(), ds);
+        } catch (IOException e) {
+            logger.log(Level.WARNING, "save profile session failed");
+            System.out.println("Failed to save your diet session!");
+        }
+    }
```

Comment:
Maybe this method should be Storage class since the Create Command should be focusing on dealing with the creation diet session